### PR TITLE
feat: login with SSO

### PR DESCRIPTION
**Pull request by pedroandrade**
- adding ability to users log in with SSO

### CODE REVIEW

File: cmd/login.go [hunk 1]
```diff
@@ -45,6 +48,42 @@ var loginCmd = &cobra.Command{
...
+			}
+
+			fmt.Println(`please login with the link below:`)
+			fmt.Printf("%s/login?device_identifier=%v\n", http.WebUrl(), deviceIdentifier)
+
+			for {
+				token, err := http.GetDeviceAuthorization(sessionToken)
+				if err != nil {
+					if errors.Is(err, http.ErrUnauthorized) {
+						log.Fatalf("We coudn't log you in, your session is expired or you not authorized to perform this action: %v", err)
```
Comment (waltzofpearls) [inline cmd/login.go:72]: `or you are not authorized`

**Review by waltzofpearls — APPROVED** [review]
👍

File: cmd/login.go (hunk 1 above)
Comment (pedroandrade) [reply cmd/login.go]: thanks rollie :D

Comment (atoonk) [inline cmd/login.go]: Can you change this text to:
We couldn't log you in, your session is expired or you are not authorized to perform this action

File: cmd/login.go [hunk 2]
```diff
@@ -45,6 +48,42 @@ var loginCmd = &cobra.Command{
...
+					}
+
+					log.Fatalf("We coudn't log you in, make sure that you properly logged in the link above: %v", err)
```
Comment (atoonk) [inline cmd/login.go:81]: change you change this to:
We couldn't log you in, make sure that you  are properly logged in using the link above

**Pull request merged**